keras/metrics.py

```diff
@@ -36,6 +36,11 @@ def sparse_categorical_accuracy(y_true, y_pred):
 def top_k_categorical_accuracy(y_true, y_pred, k=5):
     return K.mean(K.in_top_k(y_pred, K.argmax(y_true, axis=-1), k), axis=-1)
 
+
+def top_k_sparse_categorical_accuracy(y_true, y_pred, k=5):
```

Comment:
Please rename `sparse_top_k_categorical_accuracy` for consistency with other metrics names